app/components/billing/select-plan.js

```diff
@@ -133,6 +133,14 @@ export default Component.extend({
     );
   },
 
+  customSort(plans) {
+    const planIds = ['starter_plan', 'monthly_usage_plan_10k_credits', 'annual_usage_plan_130k_credits'];
+    const specificPlans = plans.filter(plan => planIds.includes(plan.id));
+    const otherPlans = plans.filter(plan => !planIds.includes(plan.id));
+
+    return [...specificPlans, ...otherPlans];
+  },
```

Comment:
I would sort by price if it is available

Reply:
Thanks - yes - startingPrice was available.
changed the logic